_data/tagging-status.yml

```diff
@@ -12489,6 +12497,51 @@
   tasks: needs tests
   updated: 2024-07-17
 
+- name: extarticle
+  type: class
+  status: unknown
+  issues:
+  tests: false
+  tasks: needs tests
+  comments: "Perhaps safer to use article + extsizes in the interim."
+  updated: 2025-07-07
+  
+- name: extbook
+  type: class
+  status: unknown
+  issues:
+  tests: false
+  tasks: needs tests
+  comments: "Perhaps safer to use book + extsizes in the interim."
+  updated: 2025-07-07
+
+- name: extletter
+  type: class
+  status: unknown
+  issues:
+  tests: false
+  tasks: needs tests
+  comments: "Perhaps safer to use letter + extsizes in the interim."
```

Comment:
letter is currently incompatible

Reply:
partially-incompatible according to status, I think leaving extletter as unknown and needs tests is OK for now. I may adjust the comment a bit